ProjectLighthouse.Servers.API/Controllers/UserEndpoints.cs

```diff
@@ -54,4 +55,32 @@ public IActionResult GetUserStatus(int id)
 
         return this.Ok(userStatus);
     }
+
+    [HttpPost("user/inviteToken")]
+    public async Task<IActionResult> CreateUserInviteToken()
+    {
+        if (Configuration.ServerConfiguration.Instance.Authentication.PrivateRegistration ||
+            Configuration.ServerConfiguration.Instance.Authentication.RegistrationEnabled)
+        {
+
+            if (!string.IsNullOrWhiteSpace(this.Request.Headers["Authorization"]))
+            {
+                string authHeader = this.Request.Headers["Authorization"].ToString().Substring(6);
```

Comment:
Why are we using `Substring` here? It just made it confusing to send the request on my end.